workspaces/sandbox/app-config.yaml

```diff
@@ -0,0 +1,117 @@
+app:
+  title: Developer Sandboxy
+  baseUrl: http://localhost:3000
+
+organization:
+  name: Red Hat
+
+sandbox:
+  recaptcha:
+    siteKey: 6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI
+
+proxy:
+  endpoints:
+    '/signup':
```

Comment:
You don't need multiple proxy endpoints for each of the API endpoints you are using. Just create one base endpoint and then make different API calls appending different endpoints.